src/modules/core/components/BannedTag/BannedTag.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { SimpleMessageValues } from '~types/index';
+import Icon from '~core/Icon';
+
+import styles from './BannedTag.css';
+
+const displayName = 'BannedTag';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  text?: MessageDescriptor | string;
+  textValues?: SimpleMessageValues };
+}
+
+const BannedTag = ({
+  children,
+  text = { id: 'label.banned' },
+  textValues,
+  ...rest
+}: Props) => {
+  return (
+    <span className={styles.themeBanned} {...rest}>
+      <Icon
+        title={{ id: 'label.banned' }}
+        name="emoji-goblin"
+        appearance={{ size: 'normal' }}
+        className={styles.icon}
+      />
+      <div className={styles.textSyles} {...rest}>
```

Comment:
It's wrong to have div inside of a span element.
Anyway, why are you recreating the markup & styles of a tag? You can just use `Tag` component & add styles, text & emoji.

Reply:
I have removed it, let me know if it's OK now! Thank you olga